packages/edit-site/src/components/header/index.js

```diff
@@ -79,6 +82,7 @@ export default function Header( {
 				'core/edit-site/toggle-list-view'
 			),
 			isVisualMode: getEditorMode() === 'visual',
+			blockEditorMode: __unstableGetEditorMode(),
```

Comment:
Maybe we should see if it makes sense to consolidate these two `__unstableGetEditorMode` and `getEditorMode`. Also by looking at this we will need to disable the button in header when we are in `code view`.

Reply:
I added a condition to disable the button on the code editor. But I agree that we should try to consolidate __unstableGetEditorMode and getEditorMode. We can add zoomed out view as a new mode to getEditorMode() it would contain 'visual' 'code', 'visual-zoomed-out', maybe navigation mode could also be 'visual-navigation' but that's another topic.